ImageEditor/src/main/kotlin/org/wordpress/android/imageeditor/preview/PreviewImageFragment.kt

```diff
@@ -4,29 +4,87 @@ import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
-
+import android.widget.ImageView.ScaleType.CENTER
+import androidx.annotation.NonNull
 import androidx.fragment.app.Fragment
 import androidx.lifecycle.ViewModelProviders
-import org.wordpress.android.imageeditor.R
+import kotlinx.android.synthetic.main.fragment_preview_image.*
+import org.wordpress.android.imageeditor.ImageEditor
+import org.wordpress.android.imageeditor.R.layout
 
 class PreviewImageFragment : Fragment() {
     private lateinit var viewModel: PreviewImageViewModel
+    private val imageEditor = ImageEditor.instance
+
+    private var wasPaused: Boolean = false
+    private var lowResImageUrl: String? = null
+
+    companion object {
+        const val ARG_LOW_RES_IMAGE_URL = "arg_low_res_image_url"
+        const val ARG_HIGH_RES_IMAGE_URL = "arg_high_res_image_url"
+    }
 
     override fun onCreateView(
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?
-    ): View? = inflater.inflate(R.layout.fragment_preview_image, container, false)
+    ): View? = inflater.inflate(layout.fragment_preview_image, container, false)
+
+    override fun onSaveInstanceState(outState: Bundle) {
+        val intent = activity?.intent
+        if (intent != null) {
```

Comment:
Do we really need to copy the values from intent into the bundle? I'd expect the system will always recreate the Activity with the same intent it was started with. Wdyt?

Reply:
Fixed here acd9e6a